CONTRIBUTING.md

```diff
@@ -105,6 +105,9 @@ git rebase -i master
 # squash squash squash
 git push -f origin
 ```
+## Failing Cucumber Scenarios
+
+If there is a failing scenario and you can not figure out why it is failing, just run the failing scenario: `bundle exec cucumber features/failing_scenario.feature:line`. By doing so Aruba will leave its set up in the `tmp/aruba` directory. Then you can run the scenario's Reek incantation there and have a better debugging.
 
```

Comment:
>> Then you can run the scenario's Reek incantation there and have a better debugging.

I am not so sure this sentence actually makes sense. Can you describe what you are trying to say here? Maybe we'll find an easier explanation :)